Source/WebCore/dom/ShadowRoot.h

```diff
@@ -77,6 +78,8 @@ class ShadowRoot final : public DocumentFragment, public TreeScope {
     bool containsFocusedElement() const { return m_containsFocusedElement; }
     void setContainsFocusedElement(bool flag) { m_containsFocusedElement = flag; }
 
+    bool isCloneable() const { return m_cloneable; }
```

Comment:
Could be nice to name the data member `m_isCloneable` too as per the style for boolean variables (needs a prefix).

Reply:
Fixed.